java/piranha/src/main/java/com/uber/piranha/XPFlagCleaner.java

```diff
@@ -356,57 +368,70 @@ private API getXPAPI(ExpressionTree et, VisitorState state) {
     return API.UNKNOWN;
   }
 
+  private Matcher<MethodInvocationTree> methodRecordMatcher(
+      MethodRecord methodRecord, VisitorState state) {
+
+    return allOf(
+        (methodRecord.isStatic() ? staticMethod().anyClass() : instanceMethod().anyClass())
+            .named(methodRecord.getMethodName()),
+        methodRecord
+            .getArgumentIdx()
+            .map(idx -> argument(idx, isArgumentMatchesFlagName(state)))
+            .orElseGet(Matchers::anything),
+        methodRecord
+            .getReceiverType()
+            .map(
+                type ->
+                    receiverOfInvocation((receiver, st) -> isSameType(type).matches(receiver, st)))
+            .orElseGet(Matchers::anything),
+        methodRecord.getReturnType().map(Matchers::isSameType).orElseGet(Matchers::anything));
+  }
+
+  private Matcher<ExpressionTree> enumConstructorArgsContainsFlagNameMatcher(
+      ImmutableCollection<PiranhaEnumRecord> enumRecords) {
+    Matcher<ExpressionTree> matchFlag =
+        (arg, stt) ->
```

Comment:
Wondering if we should just call the second argument on these two cases `state` if it's a `VisitorState`, for clarity. Or is `st` idiomatic in example Error Prone matchers? (Either way, `stt` looks like a typo to me)

Reply:
done!